pkg/backup/backuper.go

```diff
@@ -0,0 +1,326 @@
+package backup
```

Comment:
Decided to use the word `backup` instead of `snapshot`. It looks more aligned with SQLite Backup API nomenclature.

Reply:
If we decide to keep using the term backup, we should probably adjust our Notion docs to reflect that.